DUI3-DX/Connectors/Autocad/Speckle.Connectors.AutocadShared/Bindings/AutocadBasicConnectorBinding.cs

```diff
@@ -0,0 +1,137 @@
+using System.Reflection;
+using Autodesk.AutoCAD.DatabaseServices;
+using Sentry.Reflection;
+using Speckle.Connectors.Autocad.HostApp;
+using Speckle.Connectors.DUI.Bindings;
+using Speckle.Connectors.DUI.Bridge;
+using Speckle.Connectors.DUI.Models;
+using Speckle.Connectors.DUI.Models.Card;
+using Speckle.Core.Credentials;
+using Speckle.Connectors.Autocad.HostApp.Extensions;
+using Speckle.Connectors.Autocad.Utils;
+
+namespace Speckle.Connectors.Autocad.Bindings;
+
+public class AutocadBasicConnectorBinding : IBasicConnectorBinding
+{
+  public string Name { get; set; } = "baseBinding";
+  public IBridge Parent { get; }
+
+  private readonly DocumentModelStore _store;
+  private readonly AutocadSettings _settings;
+
+  public BasicConnectorBindingCommands Commands { get; }
+
+  private static Document Doc => Application.DocumentManager.MdiActiveDocument;
```

Comment:
This should come from the ConversionContext

Reply:
I see that currently we aren't using the doc from the `ConversionContext` in other connector basic bindings, and instead just calling the current active doc instead (which has its own problems, especially in rhino). Would you prefer that for Autocad, I inject the context and use the doc from there, or match the other basic bindings and call the active doc instead?